rm-main/src/ui/tabs/torrents/rustmission_torrent.rs

```diff
@@ -23,12 +25,13 @@ pub struct RustmissionTorrent {
 impl RustmissionTorrent {
     pub fn to_row(&self) -> ratatui::widgets::Row {
         Row::new([
-            self.torrent_name.as_str(),
-            self.size_when_done.as_str(),
-            self.progress.as_str(),
-            self.eta_secs.as_str(),
-            self.download_speed.as_str(),
-            self.upload_speed.as_str(),
+            Cell::new(self.torrent_name.as_str()),
+            Cell::new(""),
+            Cell::new(self.size_when_done.as_str()),
+            Cell::new(self.progress.as_str()),
+            Cell::new(self.eta_secs.as_str()),
+            Cell::new(download_speed_format(&self.download_speed)),
+            Cell::new(upload_speed_format(&self.upload_speed)),
```

Comment:
Any particular reason why these are cells now?

Reply:
ps this was from a previous compiler error, will revert